### ISSUE TITLE
Cannot remove a service's settings if entered and need to remove one.

### ISSUE THREAD

**Issue by ydkmlt84** [jorenn92/Maintainerr#1416]
**Describe the bug**
If you enter settings for, say Overseerr, but then want to remove it as an option/service, you can't. There is no way to remove it.

**Expected behavior**
Maybe a remove button or clear button or something in that area of options.

Maintainerr works fine with a blank setting if you never set up that service in the first place, as long as you don't make rules for it.

**Comment by khthe8th** [comment]
Yes, following. Recently switched from overseerr to jellyseerr and seems to have messed up maintainerr as I can't remove overseerr from my settings

**Comment by benscobie** [comment]
@khthe8th You can follow this to remove but replace tautulli with overseerr: https://github.com/jorenn92/Maintainerr/issues/1594#issuecomment-2678835650

**Comment by benscobie** [comment]
This is resolved in [v2.19](https://github.com/jorenn92/Maintainerr/releases/tag/v2.19.0)
